config/database.yml

```diff
@@ -1,20 +1,6 @@
-# If you want to change this file, please keep the changes in your working
-# copy by using
-#
-#     git update-index --skip-worktree config/database.yml
-#
-# or just use DATABASE_URL, in which case Rails will happily skip the whole
-# file.
-#
-# See https://github.com/coopdevs/timeoverflow/wiki/Keeping-your-local-files
-# for more information
-#
-
 defaults: &defaults
   adapter: postgresql
-  username: <%= ENV['DATABASE_USER'] %>  # default is null
-  collation: 'es_ES.UTF-8'
-  ctype: 'es_ES.UTF-8'
+  username: <%= ENV['DATABASE_USER'] || ENV["POSTGRES_USER"] %>
```

Comment:
out of curiosity, do we need both ENVs? can't we call the env of gh actions the same?

Reply:
Unfortunately in GH Actions containers, all PostgreSQL ENV vars should be prefixed by `POSTGRES_*`: https://docs.github.com/en/actions/using-containerized-services/creating-postgresql-service-containers (this is not necessary for our app, but for the PostgreSQL image to properly start up).

Reply:
oh I see, thanks!